packages/flutter/lib/src/rendering/custom_paint.dart

```diff
@@ -955,7 +955,7 @@ class RenderCustomPaint extends RenderProxyBox {
       config.isFocusable = properties.focusable!;
```

Comment:
we should attempt to migrate this, the focusable in SemanticsProperty should be deprecated as well